int8 cv_control_flag

float64 cv_x
float64 cv_y
float64 cv_z
float64 cv_yaw